frontend/mgramseva/lib/providers/collect_payment_provider.dart

```diff
@@ -398,7 +398,7 @@ class CollectPaymentProvider with ChangeNotifier {
                                   .toString()) +
                               ' only')),
```

Comment:
use 18 constants

Reply:
English Only Area,
where number to words translation